Altis_Life.Altis/core/actions/fn_searchAction.sqf

```diff
@@ -8,9 +8,14 @@
 params [
     ["_unit",objNull,[objNull]]
 ];
+
 if (isNull _unit) exitWith {};
+
 hint localize "STR_NOTF_Searching";
 sleep 2;
-if (player distance _unit > 5 || !alive player || !alive _unit) exitWith {hint localize "STR_NOTF_CannotSearchPerson"};
+
+if (player distance _unit > 5 || {!alive player} || {!alive _unit}) exitWith {
```

Comment:
```suggestion
if (player distance _unit > 5 || {!alive player || {!alive _unit}}) exitWith {
```
brackets always need to be like this, otherwise it doesn't make a difference in performance